content/800-data-platform/200-projects/100-create.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: 'Create projects'
+metaTitle: 'Create projects in the Prisma Data Platform'
+metaDescription: 'In the Prisma Data Platform, you can create projects from your JavaScript/TypeScript database applications that use Prisma. To do so, you need to have your database connection string at hand and configure access to the database from the Prisma Data Platform static IPs as well as select a Data Proxy region.'
+tocDepth: 1
+---
+
+<TopBlock>
+
+In the Prisma Data Platform, you can create new projects only with an available database connection string. If your database sits behind a firewall or only allows connections from specific IPs, you can enable the use of [static IPs of the platform](/data-platform/static-ips) and add them to your database allowlist. You can also select the [Data Proxy](/data-platform/data-proxy) region to prepare your project for connection pooling and load balancing in serverless environments.
+
+After you create your project, you can optionally link your Prisma schema (if you push it to a GitHub repository) so that we can keep your project in sync with its latest version.
+
+<details>
+
+<summary>Why should I link my Prisma schema and GitHub repository?</summary>
+
+The features in the [Data Browser](/data-platform/data-browser) and [Query Console](/data-platform/query-console) rely on an up-to-date schema that matches the latest data model in your database.<br /><br />
+
+When you _create a project with an existing database_, the platform introspects your database and generates a Prisma schema for your project. If you make a change to your database model and the platform does not have a linked schema, you and your team might start seeing errors while you use [Data Browser](/data-platform/data-browser) and [Query Console](/data-platform/query-console).<br /><br />
+
+If you _create a project with an empty database_, [Data Browser](/data-platform/data-browser) and [Query Console](/data-platform/query-console) remain inactive for the project because the database introspection cannot read your database model and cannot generate a Prisma schema for the project.
```

Comment:
>introspection cannot read your database model and cannot generate a Prisma schema for the project.

What I think happens is: Introspection does run and it even completes successfully for empty databases. However, the result is that there is no schema because database contains no models to read. 

I posted this in the #prisma-introspection channel, lets see what they reply and based on that maybe we can reflect the results here?

https://prisma-company.slack.com/archives/CGLQ50TK2/p1670582170886649